addons/report/models/report_paperformat.py

```diff
@@ -1,137 +1,66 @@
-                'dpi': fields.integer('Output DPI', required=True),
-                'report_ids': fields.one2many('ir.actions.report.xml',
-                                              'paperformat_id',
-                                              'Associated reports',
-                                              help="Explicitly associated reports")
-                }
-
-    def _check_format_or_page(self, cr, uid, ids, context=None):
-        for paperformat in self.browse(cr, uid, ids, context=context):
-            if paperformat.format != 'custom' and (paperformat.page_width or paperformat.page_height):
-                return False
-        return True
-
-    _constraints = [
-        (_check_format_or_page, 'Error ! You cannot select a format AND speficic '
-                                'page width/height.', ['format']),
-    ]
-
-    _defaults = {
-        'format': 'A4',
-        'margin_top': 40,
-        'margin_bottom': 20,
-        'margin_left': 7,
-        'margin_right': 7,
-        'page_height': False,
-        'page_width': False,
-        'orientation': 'Landscape',
-        'header_line': False,
-        'header_spacing': 35,
-        'dpi': 90,
-    }
-
-
-class res_company(osv.Model):
-    _inherit = 'res.company'
-
-    _columns = {'paperformat_id': fields.many2one('report.paperformat', 'Paper format')}
-
-    def init(self, cr):
-        # set a default paperformat based on rml one.
-        ref = partial(self.pool['ir.model.data'].xmlid_to_res_id, cr, SUPERUSER_ID)
-
-        ids = self.search(cr, SUPERUSER_ID, [('paperformat_id', '=', False)])
-        for company in self.browse(cr, SUPERUSER_ID, ids):
-            paperformat_id = {
-                'a4': ref('report.paperformat_euro'),
-                'us_letter': ref('report.paperformat_us'),
-            }.get(company.rml_paper_format) or ref('report.paperformat_euro')
-
-            if paperformat_id:
-                company.write({'paperformat_id': paperformat_id})
-
-        sup = super(res_company, self)
-        if hasattr(sup, 'init'):
-            sup.init(cr)
-
-
-class ir_actions_report(osv.Model):
-    _inherit = 'ir.actions.report.xml'
-
-    def associated_view(self, cr, uid, ids, context):
-        """Used in the ir.actions.report.xml form view in order to search naively after the view(s)
-        used in the rendering.
-        """
-        try:
-            report_name = self.browse(cr, uid, ids[0], context).report_name
-            act_window_obj = self.pool.get('ir.actions.act_window')
-            view_action = act_window_obj.for_xml_id(cr, uid, 'base', 'action_ui_view', context=context)
-            view_action['domain'] = [('name', 'ilike', report_name.split('.')[1]), ('type', '=', 'qweb')]
-            return view_action
-        except:
-            return False
-
-    _columns = {'paperformat_id': fields.many2one('report.paperformat', 'Paper format')}
+    name = fields.Char('Name', required=True)
```

Comment:
Not required string when string name and field name same it means in camel case like : 
```
name = fields.Char(required=True)
```